DependencyInjection/Configuration.php

```diff
@@ -465,4 +466,26 @@ public function addOptionalHeadersNode()
 
         return $node;
     }
+
+    public function addHeaderValidationNode()
+    {
+        $builder = new TreeBuilder();
+        $node = $builder->root('headerValidations');
+
+        $node
+            ->info('Add optional header validations')
```

Comment:
This is not only for optional headers, but for all headers.
Do you mind updating description?

Reply:
But the validations are optional :D I've updated it anyways to remove any confusion